docs/src/app/dithering/page.tsx

```diff
@@ -41,8 +42,8 @@ const DitheringWithControls = () => {
     return {
       Parameters: folder(
         {
-          color1: { value: defaults.color1, order: 100 },
-          color2: { value: defaults.color2, order: 101 },
+          color1: { value: toHsla(defaults.color1), order: 100 },
+          color2: { value: toHsla(defaults.color2), order: 101 },
```

Comment:
@uuuulala we'll need to add "toHsla" on pages when defining color parameters (color arrays would be handled automatically though)